src/qml/ProjectCreationScreen.qml

```diff
@@ -0,0 +1,346 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import QtQuick.Controls.Material.impl
+import org.qfield
+import Theme
+
+Page {
+  id: createNewProject
+
+  signal create(var projectConfig)
+
+  leftPadding: mainWindow.sceneLeftMargin
+  rightPadding: mainWindow.sceneRightMargin
+
+  header: QfPageHeader {
+    title: qsTr("Create a new project")
+
+    backgroundFill: false
+    showBackButton: true
+    showApplyButton: false
+    showCancelButton: false
+    showMenuButton: false
+
+    topMargin: mainWindow.sceneTopMargin
+    titleColor: Theme.mainTextColor
+    titleFont: Theme.strongTitleFont
+
+    onBack: {
+      createNewProject.visible = false;
+    }
+  }
+
+  Flickable {
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.bottom: createProjectButton.top
```

Comment:
You can go for anchors.fill: parent

What we want here is for the create project button container to sit _on top_ of the flickable view, with a semi transparent background that's opaque enough not to be distracting but transparent enough to see any content that's not in the visible part (like we do with the dashboard)